src/test/java/com/amazon/opendistroforelasticsearch/knn/plugin/action/RestKNNStatsHandlerIT.java

```diff
@@ -164,6 +170,62 @@ public void testInvalidNodeIdStats() throws Exception {
         assertEquals(0, nodeStats.size());
     }
 
+    /**
+     *  Test checks that script stats are properly updated
+     */
+    public void testScriptStats() throws Exception {
```

Comment:
Can we add similar test with multiple shards?

Reply:
Sure